security/tor/Makefile

```diff
@@ -1,7 +1,7 @@
 PLUGIN_NAME=          tor
-PLUGIN_VERSION=       1.1
+PLUGIN_VERSION=       1.2
```

Comment:
I just bumped this yesterday. Release is still at 1.0 so please make this 1.1 again

Reply:
@fichtner no - your patch goes to stable, 1.2 goes to dev

Reply:
fine, but in that case I'll leave this open for a while